docs/source/docformat.rst

```diff
@@ -4,35 +4,276 @@ Documentation Formats
+
+Fields
+^^^^^^
+
+As a reminder, here are some of the supported *restructuredtext* fields:
+
+    - ``:cvar foo:``
+    - ``:ivar foo:``
+    - ``:var foo:``
+    - ``:param bar:`` (synonym: ``:arg bar:``)
+    - ``:type bar: str``
+    - ``:return:``
+    - ``:rtype: list``
+    - ``:except ValueError:``
+
+Alternatively, fields can be passed with this syntax::
+
+    :Parameters:
+        size
+            The size of the fox (in meters)
+        weight : float
+            The weight of the fox (in stones)
+        age : int
+            The age of the fox (in years)
+
+Directives
+^^^^^^^^^^
+
+.. list-table:: Summary of supported and unsupported directives, with links to appropritate reference. 
+   :header-rows: 1
+   
+   * - Directive
+     - Reference
+     - Support
+
+   * - ``.. include::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#including-an-external-document-fragment>`_
+     - Yes
+
+   * - ``.. contents::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#table-of-contents>`_
+     - Yes
+
+   * - ``.. image::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#image>`_
+     - Yes
+       
+   * - ``.. |time| date:: %H:%M``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#date>`_
+     - Yes
+
+   * - ``.. figure::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#figure>`_
+     - Yes
+
+   * - ``.. |T| replace:: term``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#replacement-text>`_
+     - Yes
+ 
+   * - ``.. unicode::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#unicode-character-codes>`_
+     - Yes
+ 
+   * - ``.. raw::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#raw-data-pass-through>`_
+     - Yes
+  
+   * - ``.. class::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#class>`_
+     - eh
+  
+   * - ``.. role::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#custom-interpreted-text-roles>`_
+     - Yes
+  
+   * - ``.. default-role::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#setting-the-default-interpreted-text-role>`_
+     - Yes
+    
+   * - ``.. line-block::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#line-block>`_
+     - eh
+
+   * - ``.. code::``
+     - `ref (docutils) <https://docutils.sourceforge.io/docs/ref/rst/directives.html#code>`_
+     - Yes (syntax highlight ignored). Use ``.. python::``. 
+   
+   * - ``.. python::``
+     - `ref (pydoctor) <https://pydoctor.readthedocs.io/en/latest/api/pydoctor.epydoc.markup.restructuredtext.PythonCodeDirective.html>`_
```

Comment:
>  I think it would be best to link to pydoctor's version of these directives.

Some of them are not supported yet, so the links are directed to epydoc apidocs for now. For now it's unclear what will be the links to the "pydoctor reference manual" so I'll just put pydoctor as plaintext and we'll link to the quick ref once we have it published.